trimesh/visual/material.py

```diff
@@ -525,7 +525,7 @@ def normalTexture(self):
           Normal texture.
         """
         return self._data.get('normalTexture')
-    
```

Comment:
sorry for these. But I used 
```
autopep8 --recursive --verbose --in-place --aggressive trimesh
flake8 trimesh
```
as mentioned on the contributing page. But I did not check in the changes in other files. Seems like you are now using ruff instead